src/apps/emcd-swap/views/ExchangeView.tsx

```
@@ -0,0 +1,178 @@
+              interactive={true}
+            >
+              <div className='cursor-pointer'>
+                <InfoIcon />
+              </div>
+            </Tippy>
+
+            <div className="text-sm text-color-3">
+              1 {currentCoinFrom && currentCoinFrom.title}{' '}
+              <span className={'mb-1'}>≈</span>{' '}
+              <span className={'text-color-1'}>
+                {rate} {currentCoinTo && currentCoinTo.title}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)